services/gossip/adapter/tcp/queue.go

```diff
@@ -26,17 +27,21 @@ type transportQueue struct {
 		bytesLeft int
 	}
 	usagePercentageMetric *metric.Gauge
+	sendErrors            *metric.Gauge
+	sendQueueErrors       *metric.Gauge
 }
 
-func NewTransportQueue(maxSizeBytes int, maxSizeMessages int, metricFactory metric.Factory) *transportQueue {
+func NewTransportQueue(maxSizeBytes int, maxSizeMessages int, metricFactory metric.Factory, peerNodeAddress string) *transportQueue {
 	q := &transportQueue{
 		channel:     make(chan *adapter.TransportData, maxSizeMessages),
 		maxBytes:    maxSizeBytes,
 		maxMessages: maxSizeMessages,
 	}
 	q.protected.bytesLeft = maxSizeBytes
 
-	q.usagePercentageMetric = metricFactory.NewGauge("Gossip.OutgoingConnection.Queue.Usage.Percent")
+	q.usagePercentageMetric = metricFactory.NewGauge(fmt.Sprintf("Gossip.OutgoingConnection.Queue.Usage.%s.Percent", peerNodeAddress))
```

Comment:
BTW how long is the "peerNodeAddress"? is this 40 chars long?
Let's discuss this again before it goes to production. I'll want to see this on Grafana before production.

Reply:
yeah, I think it's gonna be 40 chars long